lib/lobstersbot/pluggable_connection.rb

```diff
@@ -13,14 +13,29 @@ def did_start_up
       @timers = Timers::Group.new
       @memory = PStore.new(config_dir('memory.pstore'), true)
       
+      @triggers = {}
+      # Initialize plugins
+      evaluate(:initialize)
```

Comment:
You will end up calling the constructor twice which will have some unintended side effects internal to Summer.

Reply:
`evaluate` looks for `{prefix}_`, and Summer only has `initialize` (without the `_`). The name probably isn't very good though, so what about just `init` (`init_salute` etc.)?